packages/desktop-client/src/components/accounts/AccountSyncCheck.jsx

```diff
@@ -35,6 +35,12 @@ function getErrorMessage(type, code) {
     case 'RATE_LIMIT_EXCEEDED':
       return 'Rate limit exceeded for this item. Please try again later.';
 
+    case 'INVALID_ACCESS_TOKEN':
+      return 'Your SimpleFIN Access Token is no longer valid.  Please reset and generate a new token.';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
      return 'Your SimpleFIN Access Token is no longer valid. Please reset and generate a new token.';
```